charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.2/src/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+description: |-
+  Build and run .NET applications on UBI.
+  For more information about using this builder image, including OpenShift considerations,
+  see https://github.com/redhat-developer/s2i-dotnetcore.
+annotations:
+  charts.openshift.io/name: Red Hat .NET imagestreams.
+  charts.openshift.io/provider: Red Hat
+  charts.openshift.io/providerType: redhat
+  charts.openshift.io/supportURL: https://github.com/redhat-developer/s2i-dotnetcore
+apiVersion: v2
+appVersion: "9.0"
+kubeVersion: '>=1.20.0'
+name: redhat-dotnet-imagestreams
+tags: builder,dotnet,dotnet-runtime
```

Comment:
I'm seeing the `tags` key as invalid, and should either be in an enumerated dependency (where it has some ability to impact installation behavior [[1]](https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies), or added as an annotation to the parent chart where it doesn't.

Reply:
I'll remove it. Note that we included `tags` because the other image streams charts in the repo have it as well.